app/actions/StatisticsActions.js

```diff
@@ -514,7 +657,7 @@ const prepDataToCalcStats = async (startBlock, endBlock, currentDate, endDate, p
   // grab transactions in batches of (roughly) `pageSize` transactions, so
   // that if we can stop in the middle of the process (say, because we're
   // interested in only the first 10 days worth of balances)
-  while (continueGetting) {
+  while (continueGetting !== false) {
```

Comment:
I noticed sometimes `continueGetting` was undefined, making it leave while

Reply:
Was that reproducible? It seems the only thing that could be undefined here is `backwards`, so it might be better to coerce that one into a bool at the start of the loop instead of changing the test to identity (otherwise an `undefined` backwards might not stop the iteration when it should)

Reply:
I think it happened when I was manipulating some data. Tried to reproduce but I couldn't